guides/advanced-use/build-targets.md

```diff
@@ -0,0 +1,74 @@
+Ember CLI by default uses [Babel.js](https://babeljs.io/) to allow you to use tomorrow's JavaScript, today.
+
+It will ensure that you can use the newest features in the language and know that they will be transformed to JavaScript that can run in every browser you support.
+That usually means generating ES5-compatible code that can work on any modern browser, back to Internet Explorer 11.
```

Comment:
```suggestion
That usually means generating ES5-compatible code that can work on any legacy browser, back to Internet Explorer 11.
```

Reply:
I think "modern" here is more correct, but I can see how it's confusing. How about just removing this sentence? The previous sentence ends "every browser you support", and I think that should be enough. I don't see a need to call out IE11 specifically